posts/2021-11-30-microprofile-5-22001-beta.adoc

```diff
@@ -0,0 +1,203 @@
+---
+layout: post
+title: "MicroProfile 5, MicroProfile Rest Client 3.0 and JPA enhancements in 22.0.0.1-beta"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/ryan-storey
+author_github: https://github.com/ryan-storey
+seo-title: MicroProfile 5, MicroProfile Rest Client 3.0 and JPA enhancements in 22.0.0.1-beta - OpenLiberty.io
+seo-description: Open Liberty 22.0.0.1-beta offers MicroProfile 5.0 including MicroProfile Rest Client 3.0, which align with Jakarta EE 9.1. Also available in this beta release is an enhancement to JPA which provides the ability to declare "default" JPA persistence properties. 
+blog_description: Open Liberty 22.0.0.1-beta offers MicroProfile 5.0 including MicroProfile Rest Client 3.0, which align with Jakarta EE 9.1. Also available in this beta release is an enhancement to JPA which provides the ability to declare "default" JPA persistence properties. 
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= MicroProfile 5, MicroProfile Rest Client 3.0 and JPA enhancements in Open Liberty 22.0.0.1-beta
+Ryan Storey <https://github.com/ryan-storey>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+Open Liberty 22.0.0.1-beta offers MicroProfile 5.0 including MicroProfile Rest Client 3.0, which align with Jakarta EE 9.1. Also available in this beta release is an enhancement to JPA which provides the ability to declare "default" JPA persistence properties.
```

Comment:
suggested edit:

Open Liberty 22.0.0.1-beta offers MicroProfile 5.0, which includes MicroProfile Rest Client 3.0 and aligns with Jakarta EE 9.1. This beta release also introduces the ability to declare "default" JPA persistence properties.